--- texinfo-4.0b/info/info.c~	Tue May  1 18:12:30 2001
+++ texinfo-4.0b/info/info.c	Tue Dec 11 00:18:42 2001
@@ -527,6 +527,12 @@
 static void
 info_short_help ()
 {
+#ifdef __MSDOS__
+#define SPEECH_FRIENDLY "\
+ --speech-friendly        be friendly to speech synthesizers.\n"
+#else
+#define SPEECH_FRIENDLY ""
+#endif
   printf (_("\
 Usage: %s [OPTION]... [MENU-ITEM...]\n\
 \n\
@@ -564,13 +570,7 @@
 general questions and discussion to help-texinfo@gnu.org.\n\
 "),
   program_name,
-#ifdef __MSDOS__
-"\
- --speech-friendly        be friendly to speech synthesizers.\n"
-#else
-""
-#endif
-	  );
+  SPEECH_FRIENDLY);
 
   xexit (0);
 }
