--- texinfo-4.2/info/Makefile.am~	Tue May  7 07:18:17 2002
+++ texinfo-4.2/info/Makefile.am	Tue May  7 07:18:17 2002
@@ -25,7 +25,7 @@
 # we need to override it, so include them ourselves.
 INCLUDES = -I. -I$(top_srcdir)/lib -I../intl -I.. -I$(srcdir)
 DEFS = -DINFODIR=\"$(infodir)\" -DLOCALEDIR=\"$(localedir)\" @DEFS@
-LDADD = ../lib/libtxi.a @TERMLIBS@ @LIBINTL@
+LDADD = ../lib/libtxi.a $(TERMLIBS) $(INTLLIBS)
 
 EXTRA_DIST = README pcterm.c
 
--- texinfo-4.2/util/Makefile.am~	Tue May  7 07:20:07 2002
+++ texinfo-4.2/util/Makefile.am	Tue May  7 07:20:07 2002
@@ -15,7 +15,7 @@
 
 localedir = $(datadir)/locale
 INCLUDES = -I$(top_srcdir)/lib -I../intl -DLOCALEDIR=\"$(localedir)\"
-LDADD = ../lib/libtxi.a @LIBINTL@ -lz
+LDADD = ../lib/libtxi.a $(INTLLIBS) -lz
 
 # Most of these are for fun.  The only one that's really supported is texi2dvi.
 EXTRA_DIST = README deref.c fix-info-dir fixfonts \
--- texinfo-4.2/makeinfo/Makefile.am~	Tue May  7 07:20:45 2002
+++ texinfo-4.2/makeinfo/Makefile.am	Tue May  7 07:20:45 2002
@@ -14,7 +14,7 @@
 
 localedir = $(datadir)/locale
 INCLUDES = -I$(top_srcdir)/lib -I../intl -DLOCALEDIR=\"$(localedir)\"
-LDADD = ../lib/libtxi.a @LIBINTL@
+LDADD = ../lib/libtxi.a $(INTLLIBS)
 
 makeinfo_SOURCES = \
   cmds.c cmds.h defun.c defun.h docbook.c docbook.h \
