--- texinfo-5.1/doc/info-stnd.texi.orig	2013-06-24 18:42:45.291932622 +0200
+++ texinfo-5.1/doc/info-stnd.texi	2013-06-24 18:50:22.125246783 +0200
@@ -41,10 +41,10 @@
 license to the document, as described in section 6 of the license.
 @end copying
 
-@dircategory Texinfo documentation system
+@dircategory Texinfo documentation system:
 @direntry
-* info stand-alone: (info-stnd).           Read Info documents without Emacs.
-* infokey: (info-stnd)Invoking infokey.    Compile Info customizations.
+* info stand-alone: (info-stnd).	Read Info documents without Emacs
+* infokey: (info-stnd)infokey.		Compile Info customizations
 @end direntry
 
 @titlepage
@@ -2087,12 +2087,12 @@
 textual source file using the @code{infokey} command.
 
 @menu
-* Invoking @t{infokey}::
+* @t{infokey}::
 * @t{infokey} source format::
 @end menu
 
 
-@node Invoking @t{infokey}
+@node @t{infokey}
 @section Invoking @command{infokey}
 
 @cindex invoking @command{infokey}
--- texinfo-5.1/doc/info.texi.orig	2013-06-24 18:51:07.731911536 +0200
+++ texinfo-5.1/doc/info.texi	2013-06-24 18:53:43.138571681 +0200
@@ -36,9 +36,9 @@
 @end quotation
 @end copying
 
-@dircategory Texinfo documentation system
+@dircategory Texinfo documentation system:
 @direntry
-* Info: (info).                 How to use the documentation browsing system.
+* Info: (info).				How to use the documentation browsing system
 @end direntry
 
 @titlepage
--- texinfo-5.1/doc/texinfo.txi.orig	2013-03-11 19:31:13.000000000 +0100
+++ texinfo-5.1/doc/texinfo.txi	2013-06-24 19:07:23.441870590 +0200
@@ -57,16 +57,16 @@
 @end quotation
 @end copying
 
-@dircategory Texinfo documentation system
+@dircategory Texinfo documentation system:
 @direntry
-* Texinfo: (texinfo).           The GNU documentation format.
-* install-info: (texinfo)Invoking install-info. Update info/dir entries.
-* makeinfo: (texinfo)Invoking makeinfo.         Translate Texinfo source.
-* pod2texi: (pod2texi)Invoking pod2texi.        Translate Perl POD to Texinfo.
-* texi2dvi: (texinfo)Format with texi2dvi.      Print Texinfo documents.
-* texi2pdf: (texinfo)PDF Output.                PDF output for Texinfo.
-* pdftexi2dvi: (texinfo)PDF Output.             PDF output for Texinfo.
-* texindex: (texinfo)Format with tex/texindex.  Sort Texinfo index files.
+* Texinfo: (texinfo).			The GNU documentation format
+* install-info: (texinfo)install-info.	Update info/dir entries
+* makeinfo: (texinfo)makeinfo.		Translate Texinfo source
+* pod2texi: (pod2texi)pod2texi.		Translate Perl POD to Texinfo.
+* texi2dvi: (texinfo)texi2dvi.		Print Texinfo documents
+* texi2pdf: (texinfo)texi2pdf.		PDF output for Texinfo
+* pdftexi2dvi: (texinfo)pdftexi2dvi.	PDF output for Texinfo
+* texindex: (texinfo)tex/texindex.	Sort Texinfo index files
 @end direntry
 
 @c Before release, run C-u C-c C-u C-a (texinfo-all-menus-update with a
@@ -576,8 +576,8 @@
 Formatting and Printing Hardcopy
 
 * Use @TeX{}::                     Use @TeX{} to format for hardcopy.
-* Format with @t{tex}/@t{texindex}::    How to format with explicit shell commands.
-* Format with @t{texi2dvi}::        A simpler way to format.
+* @t{tex}/@t{texindex}::                How to format with explicit shell commands.
+* @t{texi2dvi}::                    A simpler way to format.
 * Print with @t{lpr}::              How to print.
 * Within Emacs::                How to format and print from an Emacs shell.
 * Texinfo Mode Printing::       How to format and print in Texinfo mode.
@@ -590,18 +590,18 @@
 * @t{@@pagesizes}::                  How to print with customized page sizes.
 * Cropmarks and Magnification:: How to print marks to indicate the size
                                  of pages and how to print scaled up output.
-* PDF Output::                  Portable Document Format output.
+* texi2pdf::                    Portable Document Format output.
 * Obtaining @TeX{}::               How to obtain @TeX{}.
 
 @code{texi2any}: The Generic Translator for Texinfo
 
 * Reference Implementation::    @command{texi2any}: the reference implementation.
-* Invoking @t{texi2any}::           Running the translator from a shell.
+* @t{texi2any}::                    Running the translator from a shell.
 * @t{texi2any} Printed Output::     Calling @command{texi2dvi}.
 * Pointer Validation::          How to check that pointers point somewhere.
 * Customization Variables::     Configuring @command{texi2any}.
 * Internationalization of Document Strings:: Translating program-inserted text.
-* Invoking @t{pod2texi}::           Translating Perl pod to Texinfo.
+* @t{pod2texi}::                    Translating Perl pod to Texinfo.
 * @t{texi2html}::                   An ancestor of @command{texi2any}.
 
 Customization Variables
@@ -634,7 +634,7 @@
                                  located in other directories.
 * Installing Dir Entries::      How to specify what menu entry to add
                                  to the Info directory.
-* Invoking @t{install-info}::       @code{install-info} options.
+* @t{install-info}::                 @code{install-info} options.
 
 Generating HTML
 
@@ -974,7 +974,7 @@
 @uref{http://tug.org/TUGboat/tb22-3/tb72beebe-pdf.pdf} have some
 background.  By default, Texinfo uses the @command{pdftex} program, a
 variant of @TeX{}, to output PDF; see
-@uref{http://tug.org/applications/pdftex}.  @xref{PDF Output}.
+@uref{http://tug.org/applications/pdftex}.  @xref{texi2pdf}.
 
 @item Docbook
 @cindex Docbook output, overview
@@ -2546,8 +2546,7 @@
 this, you must run the @code{texindex} command after first running the
 @code{tex} typesetting command; and then you must run the @code{tex}
 command again.  Or else run the @code{texi2dvi} command which
-automatically creates indices as needed (@pxref{Format with
-@t{texi2dvi}}).
+automatically creates indices as needed (@pxref{@t{texi2dvi}}).
 
 Often, when you are writing a document, you want to typeset and print
 only part of a file to see what it will look like.  You can use the
@@ -3067,7 +3066,7 @@
 when generating Info, etc.  The @code{\input} line is still ignored in
 this processing, as well as leading blank lines.
 
-See also the @option{--output} option in @ref{Invoking @t{texi2any}}.
+See also the @option{--output} option in @ref{@t{texi2any}}.
 
 
 @node @t{@@settitle}
@@ -3699,8 +3698,8 @@
 
 You need to include the @code{@@set@dots{}contentsaftertitlepage}
 commands early in the document (just after @code{@@setfilename}, for
-example).  We recommend using @command{texi2dvi} (@pxref{Format with
-@t{texi2dvi}}) to specify this without altering the source file at
+example).  We recommend using @command{texi2dvi} (@pxref{@t{texi2dvi}})
+to specify this without altering the source file at
 all.  For example:
 
 @example
@@ -13688,7 +13687,7 @@
 extended @TeX{} programs @command{etex} (DVI output) and
 @command{pdftex} (PDF output), such as @TeX{} Live 2008.
 @command{texi2dvi} will use the extended @TeX{}s if they are available
-(@pxref{Format with @t{texi2dvi}}).
+(@pxref{@t{texi2dvi}}).
 
 In September 2006, the W3C Internationalization Activity released a
 new recommendation for specifying languages:
@@ -14962,8 +14961,8 @@
 some of the caveats are listed below.  Thus, if you get macro-related
 errors when producing the printed version of a manual, you might try
 expanding the macros with @command{makeinfo} by invoking
-@command{texi2dvi} with the @samp{-E} option (@pxref{Format with
-@t{texi2dvi}}).  Or, more reliably, eschew Texinfo macros altogether
+@command{texi2dvi} with the @samp{-E} option (@pxref{@t{texi2dvi}}).
+Or, more reliably, eschew Texinfo macros altogether
 and use a language designed for macro processing, such as M4
 (@pxref{External Macro Processors}).
 
@@ -15377,8 +15376,8 @@
 
 @itemize @bullet
 @item
-If you run @command{texi2dvi} or its variants (@pxref{Format with
-@t{texi2dvi}}), you can pass @option{-E} and @command{texi2dvi}
+If you run @command{texi2dvi} or its variants (@pxref{@t{texi2dvi}}),
+you can pass @option{-E} and @command{texi2dvi}
 will run @command{makeinfo} first to expand macros and eliminate
 @samp{#line}.
 
@@ -15743,8 +15742,8 @@
 
 @menu
 * Use @TeX{}::                     Use @TeX{} to format for hardcopy.
-* Format with @t{tex}/@t{texindex}::    How to format with explicit shell commands.
-* Format with @t{texi2dvi}::        A simpler way to format.
+* @t{tex}/@t{texindex}::                How to format with explicit shell commands.
+* @t{texi2dvi}::                    A simpler way to format.
 * Print with @t{lpr}::              How to print.
 * Within Emacs::                How to format and print from an Emacs shell.
 * Texinfo Mode Printing::       How to format and print in Texinfo mode.
@@ -15757,7 +15756,7 @@
 * @t{@@pagesizes}::                  How to print with customized page sizes.
 * Cropmarks and Magnification:: How to print marks to indicate the size
                                  of pages and how to print scaled up output.
-* PDF Output::                  Portable Document Format output.
+* texi2pdf::                    Portable Document Format output.
 * Obtaining @TeX{}::               How to obtain @TeX{}.
 @end menu
 
@@ -15774,7 +15773,7 @@
 itself.
 
 
-@node Format with @t{tex}/@t{texindex}
+@node @t{tex}/@t{texindex}
 @section Format with @code{tex}/@code{texindex}
 
 @cindex Shell formatting with @code{tex} and @code{texindex}
@@ -15874,7 +15873,7 @@
 
 @pindex texi2dvi
 Alternatively, it's a one-step process: run @code{texi2dvi}
-(@pxref{Format with @t{texi2dvi}}).
+(@pxref{@t{texi2dvi}}).
 
 You need not run @code{texindex} each time after you run @code{tex}.  If
 you do not, on the next run, the @code{tex} formatting command will use
@@ -15906,7 +15905,7 @@
 (@pxref{Pointer Validation}).
 
 
-@node Format with @t{texi2dvi}
+@node @t{texi2dvi}
 @section Format with @code{texi2dvi}
 
 @pindex texi2dvi @r{(shell script)}
@@ -15945,7 +15944,7 @@
 @opindex --pdf@r{, for @command{texi2dvi}}
 @pindex pdftexi2dvi
 With the @option{--pdf} option, @command{texi2dvi} produces PDF output
-instead of DVI (@pxref{PDF Output}), by running @command{pdftex}
+instead of DVI (@pxref{texi2pdf}), by running @command{pdftex}
 instead of @command{tex}.  Alternatively, the command
 @command{texi2pdf} is an abbreviation for running @samp{texi2dvi
 --pdf}.  The command @command{pdftexi2dvi} is also supported as a
@@ -16551,7 +16550,7 @@
 @xref{@t{@@small@dots{}}}, for information about commands that make
 it easier to produce examples for a smaller manual.
 
-@xref{Format with @t{texi2dvi}}, and @ref{Preparing for @TeX{}},
+@xref{@t{texi2dvi}}, and @ref{Preparing for @TeX{}},
 for other ways to format with @code{@@smallbook} that do not require
 changing the source file.
 
@@ -16581,7 +16580,7 @@
 @end group
 @end example
 
-@xref{Format with @t{texi2dvi}}, and @ref{Preparing for @TeX{}},
+@xref{@t{texi2dvi}}, and @ref{Preparing for @TeX{}},
 for other ways to format for different paper sizes that do not require
 changing the source file.
 
@@ -16630,7 +16629,7 @@
 margins, you must define a new command in @file{texinfo.tex} or
 @file{texinfo.cnf}.
 
-@xref{Format with @t{texi2dvi}}, and @ref{Preparing for @TeX{}},
+@xref{@t{texi2dvi}}, and @ref{Preparing for @TeX{}},
 for other ways to specify @code{@@pagesizes} that do not require
 changing the source file.
 
@@ -16698,7 +16697,7 @@
 magnifications.  Be prepared to experiment.
 
 
-@node PDF Output
+@node texi2pdf
 @section PDF Output
 @cindex PDF output
 
@@ -16706,7 +16705,7 @@
 The simplest way to generate PDF output from Texinfo source is to run
 the convenience script @command{texi2pdf} (or @command{pdftexi2dvi});
 this executes the @command{texi2dvi} script with the @option{--pdf}
-option (@pxref{Format with @t{texi2dvi}}).  If for some reason you
+option (@pxref{@t{texi2dvi}}).  If for some reason you
 want to process the document by hand, you can run the @command{pdftex}
 program instead of plain @command{tex}.  That is, run @samp{pdftex
 foo.texi} instead of @samp{tex foo.texi}.
@@ -16790,12 +16789,12 @@
 
 @menu
 * Reference Implementation::    @command{texi2any}: the reference implementation.
-* Invoking @t{texi2any}::           Running the translator from a shell.
+* @t{texi2any}::                    Running the translator from a shell.
 * @t{texi2any} Printed Output::     Calling @command{texi2dvi}.
 * Pointer Validation::          How to check that pointers point somewhere.
 * Customization Variables::     Configuring @command{texi2any}.
 * Internationalization of Document Strings:: Translating program-inserted text.
-* Invoking @t{pod2texi}::           Translating Perl pod to Texinfo.
+* @t{pod2texi}::                    Translating Perl pod to Texinfo.
 * @t{texi2html}::                   An ancestor of @command{texi2any}.
 @end menu
 
@@ -16856,10 +16855,10 @@
 @command{texi2any}.
 
 
-@node Invoking @t{texi2any}
+@node @t{texi2any}
 @section Invoking @command{texi2any}/@code{makeinfo} from a Shell
 
-@anchor{Invoking makeinfo}
+@anchor{makeinfo}
 @pindex makeinfo
 @pindex texi2any
 
@@ -17383,8 +17383,8 @@
 requested.
 
 The output format options for this are @option{--dvi},
-@option{--dvipdf}, @option{--pdf}, and @option{--ps}.  @xref{Format
-with @t{texi2dvi}}, for more details on these options and general
+@option{--dvipdf}, @option{--pdf}, and @option{--ps}.  @xref{@t{texi2dvi}},
+for more details on these options and general
 @command{texi2dvi} operation.  In addition, the @option{--verbose},
 @option{--silent}, and @option{--quiet} options are passed on if
 specified; the @option{-I} and @option{-o} options are likewise passed
@@ -17554,7 +17553,7 @@
 @cindex Options, customization variables for
 
 The following table gives the customization variables associated with
-some command line options.  @xref{Invoking @t{texi2any}}, for the
+some command line options.  @xref{@t{texi2any}}, for the
 meaning of the options.
 
 @multitable @columnfractions 0.5 0.5
@@ -17629,7 +17628,7 @@
 @itemx xml
 These correspond to the command-line options (and
 @code{TEXINFO_OUTPUT_FORMAT} environment variable values) of the same
-name.  @xref{Invoking @t{texi2any}}.
+name.  @xref{@t{texi2any}}.
 
 @item debugcount
 Instead of generating a regular output format, output the count of
@@ -18432,11 +18431,11 @@
 appropriately in many languages.
 
 
-@node Invoking @t{pod2texi}
+@node @t{pod2texi}
 @section Invoking @t{pod2texi}: 
 
 @pindex pod2texi
-@cindex Invoking @code{pod2texi}
+@cindex @code{pod2texi}
 @cindex POD, converting to Texinfo
 @cindex Perl POD, converting to Texinfo
 
@@ -18884,7 +18883,7 @@
                                  located in other directories.
 * Installing Dir Entries::      How to specify what menu entry to add
                                  to the Info directory.
-* Invoking @t{install-info}::       @code{install-info} options.
+* @t{install-info}::                @code{install-info} options.
 @end menu
 
 
@@ -19122,7 +19121,7 @@
 @@dircategory Texinfo documentation system
 @@direntry
 * Texinfo: (texinfo).           The GNU documentation format.
-* install-info: (texinfo)Invoking install-info. @dots{}
+* install-info: (texinfo)install-info. @dots{}
 @dots{}
 @@end direntry
 @end smallexample
@@ -19133,7 +19132,7 @@
 INFO-DIR-SECTION Texinfo documentation system
 START-INFO-DIR-ENTRY
 * Texinfo: (texinfo).           The GNU documentation format.
-* install-info: (texinfo)Invoking install-info. @dots{}
+* install-info: (texinfo)install-info. @dots{}
 @dots{}
 END-INFO-DIR-ENTRY
 @end smallexample
@@ -19155,7 +19154,7 @@
 most others.  Description for individual utilities best start in
 column 48, where possible.  For more information about formatting see
 the @samp{--calign}, @samp{--align}, and @samp{--max-width} options in
-@ref{Invoking @t{install-info}}.
+@ref{@t{install-info}}.
 
 If you use @code{@@dircategory} more than once in the Texinfo source,
 each usage specifies the `current' category; any subsequent
@@ -19190,7 +19189,7 @@
 traditional @command{man} system.
 
 
-@node Invoking @t{install-info}
+@node @t{install-info}
 @subsection Invoking @command{install-info}
 
 @pindex install-info
@@ -19481,7 +19480,7 @@
 (@code{thead}, @code{abbr}, @code{acronym}).
 
 Using @samp{--init-file=html32.pm} produces strict HTML@tie{}3.2
-output (@pxref{Invoking @t{texi2any}}).
+output (@pxref{@t{texi2any}}).
 
 Please report output from an error-free run of @code{makeinfo} which
 has browser portability problems as a bug (@pxref{Reporting Bugs}).
@@ -19512,7 +19511,7 @@
 @code{CASE_INSENSITIVE_FILENAMES}.
 
 It is also possible to split at chapters or sections with
-@option{--split} (@pxref{Invoking @t{texi2any}}).  In that case,
+@option{--split} (@pxref{@t{texi2any}}).  In that case,
 the file names are constructed after the name of the node associated
 with the relevant sectioning command.  Also, unless
 @option{--no-node-files} is specified, a redirection file is output
