diff -ur texinfo-4.1-/doc/info-stnd.texi texinfo-4.1/doc/info-stnd.texi
--- texinfo-4.1-/doc/info-stnd.texi	Sat Mar  2 16:03:54 2002
+++ texinfo-4.1/doc/info-stnd.texi	Sat Mar  9 15:58:09 2002
@@ -10,10 +10,10 @@
 
 @include version-stnd.texi
 
-@dircategory Texinfo documentation system
+@dircategory Texinfo documentation system:
 @direntry
-* info standalone: (info-stnd).            Read Info documents without Emacs.
-* infokey: (info-stnd)Invoking infokey.    Compile Info customizations.
+* info standalone: (info-stnd).		Read Info documents without Emacs.
+* infokey: (info-stnd)Invoking infokey.	Compile Info customizations.
 @end direntry
 
 @ifinfo
Only in texinfo-4.1/doc: info-stnd.texi.orig
Only in texinfo-4.1/doc: info-stnd.texi.rej
Only in texinfo-4.1/doc: info.texi.orig
Only in texinfo-4.1/doc: texinfo.txi.orig
--- texinfo-4.2/doc/info.texi.orig	Mon Mar 18 10:00:32 2002
+++ texinfo-4.2/doc/info.texi	Tue May  7 06:41:06 2002
@@ -8,9 +8,10 @@
 @comment %**end of header
 @comment $Id$
 
-@dircategory Texinfo documentation system
+@dircategory Texinfo documentation system:
 @direntry
-* Info: (info).         How to use the documentation browsing system.
+* Info: (info).				How to use the documentation
+					browsing system
 @end direntry
 
 @ifinfo
diff -ur texinfo-4.1-/doc/texinfo.txi texinfo-4.1/doc/texinfo.txi
--- texinfo-4.1-/doc/texinfo.txi	Mon Mar  4 15:52:53 2002
+++ texinfo-4.1/doc/texinfo.txi	Sat Mar  9 15:55:55 2002
@@ -22,13 +22,13 @@
 
 @comment %**end of header
 
-@dircategory Texinfo documentation system
+@dircategory Texinfo documentation system:
 @direntry
-* Texinfo: (texinfo).           The GNU documentation format.
-* install-info: (texinfo)Invoking install-info. Update info/dir entries.
-* texi2dvi: (texinfo)Format with texi2dvi.      Print Texinfo documents.
-* texindex: (texinfo)Format with tex/texindex.  Sort Texinfo index files.
-* makeinfo: (texinfo)Invoking makeinfo.         Translate Texinfo source.
+* Texinfo: (texinfo)			The GNU documentation format
+* install-info: (texinfo)install-info	Update info/dir entries
+* texi2dvi: (texinfo)texi2dvi		Print Texinfo documents
+* texindex: (texinfo)texindex		Sort Texinfo index files
+* makeinfo: (texinfo)makeinfo		Translate Texinfo source
 @end direntry
 
 @c Before release, run C-u C-c C-u C-a (texinfo-all-menus-update with a
@@ -517,8 +517,8 @@
 Formatting and Printing Hardcopy
 
 * Use TeX::                     Use @TeX{} to format for hardcopy.
-* Format with tex/texindex::    How to format with explicit shell commands.
-* Format with texi2dvi::        A simpler way to format.
+* texindex::			How to format with explicit shell commands.
+* texi2dvi::			A simpler way to format.
 * Print with lpr::              How to print.
 * Within Emacs::                How to format and print from an Emacs shell.
 * Texinfo Mode Printing::       How to format and print in Texinfo mode.
@@ -561,7 +561,7 @@
                                   located in other directories.
 * Installing Dir Entries::      How to specify what menu entry to add
                                   to the Info directory.
-* Invoking install-info::       @code{install-info} options.
+* install-info::		@code{install-info} options.
 
 Include Files
 
@@ -583,7 +583,7 @@
 
 Formatting Mistakes
 
-* makeinfo Preferred::          @code{makeinfo} finds errors.
+* makeinfo::			@code{makeinfo} finds errors.
 * Debugging with Info::         How to catch errors with Info formatting.
 * Debugging with TeX::          How to catch errors with @TeX{} formatting.
 * Using texinfo-show-structure::  How to use @code{texinfo-show-structure}.
@@ -2263,7 +2263,7 @@
 you must run the @code{texindex} command after first running the
 @code{tex} typesetting command; and then you must run the @code{tex}
 command again.  Or else run the @code{texi2dvi} command which
-automatically creates indices as needed (@pxref{Format with texi2dvi}).
+automatically creates indices as needed (@pxref{texi2dvi}).
 
 Often, when you are writing a document, you want to typeset and print
 only part of a file to see what it will look like.  You can use the
@@ -3970,8 +3970,8 @@
 
 You need to include the @code{@@set@dots{}contentsaftertitlepage}
 commands early in the document (just after @code{@@setfilename}, for
-example).  We recommend using @command{texi2dvi} (@pxref{Format with
-texi2dvi}) to specify this without altering the source file at all.  For
+example).  We recommend using @command{texi2dvi} (@pxref{(texinfo)texi2dvi})
+to specify this without altering the source file at all.  For
 example:
 @example
 texi2dvi --texinfo=@@setcontentsaftertitlepage foo.texi
@@ -12680,8 +12680,7 @@
 
 If some macro feature causes errors when producing the printed version
 of a manual, try expanding the macros with @command{makeinfo} by
-invoking @command{texi2dvi} with the @samp{-e} option; see @ref{Format
-with texi2dvi}.
+invoking @command{texi2dvi} with the @samp{-e} option; see @ref{texi2dvi}.
 
 @node alias
 @section @samp{@@alias @var{new}=@var{existing}}
@@ -12825,8 +12825,8 @@
 
 @menu
 * Use TeX::                     Use @TeX{} to format for hardcopy.
-* Format with tex/texindex::    How to format with explicit shell commands.
-* Format with texi2dvi::        A simpler way to format.
+* texindex::			How to format with explicit shell commands.
+* texi2dvi::			A simpler way to format.
 * Print with lpr::              How to print.
 * Within Emacs::                How to format and print from an Emacs shell.
 * Texinfo Mode Printing::       How to format and print in Texinfo mode.
@@ -12857,7 +12857,7 @@
 File}).
 
 
-@node Format with tex/texindex
+@node texindex
 @section Format with @code{tex} and @code{texindex}
 @cindex Shell formatting with @code{tex} and @code{texindex}
 @cindex Formatting with @code{tex} and @code{texindex}
@@ -12878,7 +12878,7 @@
 @pindex texindex
 The @code{tex} formatting command itself does not sort the indices; it
 writes an output file of unsorted index data.  (The @code{texi2dvi}
-command automatically generates indices; @pxref{Format with texi2dvi,,
+command automatically generates indices; @pxref{texi2dvi,,
 Format with @code{texi2dvi}}.)  To generate a printed index after
 running the @code{tex} command, you first need a sorted index to work
 from.  The @code{texindex} command sorts indices.  (The source file
@@ -12955,7 +12955,7 @@
 
 @pindex texi2dvi
 Alternatively, it's a one-step process: run @code{texi2dvi}
-(@pxref{Format with texi2dvi}).
+(@pxref{texi2dvi}).
 
 You need not run @code{texindex} each time after you run @code{tex}.  If
 you do not, on the next run, the @code{tex} formatting command will use
@@ -12987,7 +12987,7 @@
 (@pxref{Pointer Validation}).
 
 
-@node Format with texi2dvi
+@node texi2dvi
 @section Format with @code{texi2dvi}
 @pindex texi2dvi @r{(shell script)}
 
@@ -13532,7 +13532,7 @@
 @xref{small}, for information about
 commands that make it easier to produce examples for a smaller manual.
 
-@xref{Format with texi2dvi}, and @ref{Preparing for TeX,,Preparing for
+@xref{texi2dvi}, and @ref{Preparing for TeX,,Preparing for
 @TeX{}}, for other ways to format with @code{@@smallbook} that do not
 require changing the source file.
 
@@ -13562,7 +13562,7 @@
 @end group
 @end example
 
-@xref{Format with texi2dvi}, and @ref{Preparing for TeX,,Preparing for
+@xref{texi2dvi}, and @ref{Preparing for TeX,,Preparing for
 @TeX{}}, for other ways to format for different paper sizes that do not
 require changing the source file.
 
@@ -13608,7 +13608,7 @@
 margins, you must define a new command in @file{texinfo.tex} (or
 @file{texinfo.cnf}, @pxref{Preparing for TeX,,Preparing for @TeX{}}).
 
-@xref{Format with texi2dvi}, and @ref{Preparing for TeX,,Preparing for
+@xref{texi2dvi}, and @ref{Preparing for TeX,,Preparing for
 @TeX{}}, for other ways to specify @code{@@pagesizes} that do not
 require changing the source file.
 
@@ -14453,7 +14453,7 @@
                                   located in other directories.
 * Installing Dir Entries::      How to specify what menu entry to add
                                   to the Info directory.
-* Invoking install-info::       @code{install-info} options.
+* install-info::		@code{install-info} options.
 @end menu
 
 
@@ -14700,7 +14700,7 @@
 @@dircategory Texinfo documentation system
 @@direntry
 * Texinfo: (texinfo).           The GNU documentation format.
-* install-info: (texinfo)Invoking install-info. @dots{}
+* install-info: (texinfo)install-info. @dots{}
 @dots{}
 @@end direntry
 @end smallexample
@@ -14711,7 +14711,7 @@
 INFO-DIR-SECTION Texinfo documentation system
 START-INFO-DIR-ENTRY
 * Texinfo: (texinfo).           The GNU documentation format.
-* install-info: (texinfo)Invoking install-info. @dots{}
+* install-info: (texinfo)install-info. @dots{}
 @dots{}
 END-INFO-DIR-ENTRY
 @end smallexample
@@ -14747,8 +14747,8 @@
 as a whole in the appropriate other category.
 
 
-@node Invoking install-info
-@subsection Invoking install-info
+@node install-info
+@subsection install-info
 
 @pindex install-info
 
@@ -17143,7 +17143,7 @@
 command and you can use the @kbd{M-x Info-validate} command.@refill
 
 @menu
-* makeinfo Preferred::          @code{makeinfo} finds errors.
+* makeinfo::			@code{makeinfo} finds errors.
 * Debugging with Info::         How to catch errors with Info formatting.
 * Debugging with TeX::          How to catch errors with @TeX{} formatting.
 * Using texinfo-show-structure::  How to use @code{texinfo-show-structure}.
@@ -17151,7 +17151,7 @@
 * Running Info-Validate::       How to find badly referenced nodes.
 @end menu
 
-@node makeinfo Preferred, Debugging with Info, Catching Mistakes, Catching Mistakes
+@node makeinfo, Debugging with Info, Catching Mistakes, Catching Mistakes
 @ifinfo
 @heading @code{makeinfo} Find Errors
 @end ifinfo
@@ -17170,7 +17170,7 @@
 cannot use @code{makeinfo}, or your problem is very puzzling, then you
 may want to use the tools described in this appendix.@refill
 
-@node Debugging with Info, Debugging with TeX, makeinfo Preferred, Catching Mistakes
+@node Debugging with Info, Debugging with TeX, makeinfo, Catching Mistakes
 @comment  node-name,  next,  previous,  up
 @section Catching Errors with Info Formatting
 @cindex Catching errors with Info formatting
