diff -dur texinfo-4.13.orig/configure.ac texinfo-4.13/configure.ac
--- texinfo-4.13.orig/configure.ac	2012-12-06 09:11:23.000000000 +0100
+++ texinfo-4.13/configure.ac	2012-12-06 09:11:54.000000000 +0100
@@ -13,7 +13,7 @@
 
 dnl Must come before AM_INIT_AUTOMAKE.
 AC_CONFIG_AUX_DIR([build-aux])
-AM_INIT_AUTOMAKE([1.10.1 readme-alpha dist-lzma])
+AM_INIT_AUTOMAKE([1.10.1 readme-alpha])
 
 # Minimum Autoconf version required.
 AC_PREREQ(2.59)
