--- texinfo-4.7/doc/info-stnd.texi.orig	2004-03-24 02:41:06.000000000 +0100
+++ texinfo-4.7/doc/info-stnd.texi	2004-04-12 13:07:58.150558008 +0200
@@ -35,10 +35,10 @@
 @end quotation
 @end copying
 
-@dircategory Texinfo documentation system
+@dircategory Texinfo documentation system:
 @direntry
-* info standalone: (info-stnd).            Read Info documents without Emacs.
-* infokey: (info-stnd)Invoking infokey.    Compile Info customizations.
+* info standalone: (info-stnd).		Read Info documents without Emacs
+* infokey: (info-stnd)infokey.		Compile Info customizations
 @end direntry
 
 @titlepage
@@ -1963,12 +1963,12 @@
 textual source file using the @code{infokey} command.
 
 @menu
-* Invoking infokey::
+* infokey::
 * infokey source format::
 @end menu
 
 
-@node Invoking infokey
+@node infokey
 @section Invoking @command{infokey}
 
 @cindex invoking infokey
--- texinfo-4.7/doc/info.texi.orig	2004-04-06 10:21:35.000000000 +0200
+++ texinfo-4.7/doc/info.texi	2004-04-12 13:08:41.859913176 +0200
@@ -37,9 +37,9 @@
 @end quotation
 @end copying
 
-@dircategory Texinfo documentation system
+@dircategory Texinfo documentation system:
 @direntry
-* Info: (info).         How to use the documentation browsing system.
+* Info: (info).				The documentation browsing system
 @end direntry
 
 @titlepage
--- texinfo-4.11/doc/texinfo.txi.orig	2007-07-29 16:55:43.000000000 +0200
+++ texinfo-4.11/doc/texinfo.txi	2007-11-01 01:47:07.995421306 +0100
@@ -56,15 +56,15 @@
 @end quotation
 @end copying
 
-@dircategory Texinfo documentation system
+@dircategory Texinfo documentation system:
 @direntry
-* Texinfo: (texinfo).           The GNU documentation format.
-* install-info: (texinfo)Invoking install-info. Update info/dir entries.
-* texi2dvi: (texinfo)Format with texi2dvi.      Print Texinfo documents.
-* texi2pdf: (texinfo)PDF Output.                PDF output for Texinfo.
-* pdftexi2dvi: (texinfo)PDF Output.             PDF output for Texinfo.
-* texindex: (texinfo)Format with tex/texindex.  Sort Texinfo index files.
-* makeinfo: (texinfo)Invoking makeinfo.         Translate Texinfo source.
+* Texinfo: (texinfo).			The GNU documentation format
+* install-info: (texinfo)install-info.	Update info/dir entries
+* texi2dvi: (texinfo)texi2dvi.		Print Texinfo documents
+* texi2pdf: (texinfo)texi2pdf.		PDF output for Texinfo
+* pdftexi2dvi: (texinfo)pdftexi2dvi.    PDF output for Texinfo
+* texindex: (texinfo)texindex.		Sort Texinfo index files
+* makeinfo: (texinfo)makeinfo.		Translate Texinfo source
 @end direntry
 
 @c Before release, run C-u C-c C-u C-a (texinfo-all-menus-update with a
@@ -566,8 +566,8 @@
 Formatting and Printing Hardcopy
 
 * Use TeX::                     Use @TeX{} to format for hardcopy.
-* Format with tex/texindex::    How to format with explicit shell commands.
-* Format with texi2dvi::        A simpler way to format.
+* texindex::                    How to format with explicit shell commands.
+* texi2dvi::                    A simpler way to format.
 * Print with lpr::              How to print.
 * Within Emacs::                How to format and print from an Emacs shell.
 * Texinfo Mode Printing::       How to format and print in Texinfo mode.
@@ -580,7 +580,7 @@
 * pagesizes::                   How to print with customized page sizes.
 * Cropmarks and Magnification:: How to print marks to indicate the size
                                  of pages and how to print scaled up output.
-* PDF Output::                  Portable Document Format output.
+* texi2pdf::                    Portable Document Format output.
 * Obtaining TeX::               How to Obtain @TeX{}.
 
 Creating and Installing Info Files
@@ -591,7 +591,7 @@
 Creating an Info File
 
 * makeinfo advantages::         @code{makeinfo} provides better error checking.
-* Invoking makeinfo::           How to run @code{makeinfo} from a shell.
+* makeinfo::                    How to run @code{makeinfo} from a shell.
 * makeinfo options::            Specify fill-column and other options.
 * Pointer Validation::          How to check that pointers point somewhere.
 * makeinfo in Emacs::           How to run @code{makeinfo} from Emacs.
@@ -610,7 +610,7 @@
                                  located in other directories.
 * Installing Dir Entries::      How to specify what menu entry to add
                                  to the Info directory.
-* Invoking install-info::       @code{install-info} options.
+* install-info::                @code{install-info} options.
 
 Generating HTML
 
@@ -905,7 +905,7 @@
 @uref{http://tug.org/TUGboat/Articles/tb22-3/tb72beebe-pdf.pdf} has
 some background.  Texinfo uses the @command{pdftex} program, a variant
 of @TeX{}, to output PDF; see
-@uref{http://tug.org/applications/pdftex}.  @xref{PDF Output}.
+@uref{http://tug.org/applications/pdftex}.  @xref{texi2pdf}.
 
 @item XML
 @cindex XML output
@@ -2415,7 +2415,7 @@
 you must run the @code{texindex} command after first running the
 @code{tex} typesetting command; and then you must run the @code{tex}
 command again.  Or else run the @code{texi2dvi} command which
-automatically creates indices as needed (@pxref{Format with texi2dvi}).
+automatically creates indices as needed (@pxref{texi2dvi}).
 
 Often, when you are writing a document, you want to typeset and print
 only part of a file to see what it will look like.  You can use the
@@ -3616,8 +3616,8 @@
 
 You need to include the @code{@@set@dots{}contentsaftertitlepage}
 commands early in the document (just after @code{@@setfilename}, for
-example).  We recommend using @command{texi2dvi} (@pxref{Format with
-texi2dvi}) to specify this without altering the source file at all.  For
+example).  We recommend using @command{texi2dvi} (@pxref{texi2dvi})
+to specify this without altering the source file at all.  For
 example:
 @example
 texi2dvi --texinfo=@@setcontentsaftertitlepage foo.texi
@@ -13968,7 +13968,7 @@
 problems with @TeX{}.  If you get macro-related errors when producing
 the printed version of a manual, try expanding the macros with
 @command{makeinfo} by invoking @command{texi2dvi} with the @samp{-E}
-option (@pxref{Format with texi2dvi}).
+option (@pxref{texi2dvi}).
 
 @itemize @bullet
 @item
@@ -14237,8 +14237,8 @@
 
 @menu
 * Use TeX::                     Use @TeX{} to format for hardcopy.
-* Format with tex/texindex::    How to format with explicit shell commands.
-* Format with texi2dvi::        A simpler way to format.
+* texindex::                    How to format with explicit shell commands.
+* texi2dvi::                    A simpler way to format.
 * Print with lpr::              How to print.
 * Within Emacs::                How to format and print from an Emacs shell.
 * Texinfo Mode Printing::       How to format and print in Texinfo mode.
@@ -14251,7 +14251,7 @@
 * pagesizes::                   How to print with customized page sizes.
 * Cropmarks and Magnification:: How to print marks to indicate the size
                                  of pages and how to print scaled up output.
-* PDF Output::                  Portable Document Format output.
+* texi2pdf::                    Portable Document Format output.
 * Obtaining TeX::               How to Obtain @TeX{}.
 @end menu
 
@@ -14270,7 +14270,7 @@
 File}).
 
 
-@node Format with tex/texindex
+@node texindex
 @section Format with @code{tex} and @code{texindex}
 @cindex Shell formatting with @code{tex} and @code{texindex}
 @cindex Formatting with @code{tex} and @code{texindex}
@@ -14367,7 +14367,7 @@
 
 @pindex texi2dvi
 Alternatively, it's a one-step process: run @code{texi2dvi}
-(@pxref{Format with texi2dvi}).
+(@pxref{texi2dvi}).
 
 You need not run @code{texindex} each time after you run @code{tex}.  If
 you do not, on the next run, the @code{tex} formatting command will use
@@ -14399,7 +14399,7 @@
 (@pxref{Pointer Validation}).
 
 
-@node Format with texi2dvi
+@node texi2dvi
 @section Format with @code{texi2dvi}
 @pindex texi2dvi @r{(shell script)}
 
@@ -14436,7 +14436,7 @@
 
 @opindex --pdf @r{(@command{texi2dvi})}
 With the @option{--pdf} option, @command{texi2dvi} produces PDF output
-instead of DVI (@pxref{PDF Output}), by running @command{pdftex}
+instead of DVI (@pxref{texi2pdf}), by running @command{pdftex}
 instead of @command{tex}.  Alternatively, the command
 @command{texi2pdf} is an abbreviation for running @samp{texi2dvi
 --pdf}.  The command @command{pdftexi2dvi} is also supported as a
@@ -15013,7 +15013,7 @@
 @xref{small}, for information about
 commands that make it easier to produce examples for a smaller manual.
 
-@xref{Format with texi2dvi}, and @ref{Preparing for TeX,,Preparing for
+@xref{texi2dvi}, and @ref{Preparing for TeX,,Preparing for
 @TeX{}}, for other ways to format with @code{@@smallbook} that do not
 require changing the source file.
 
@@ -15043,7 +15043,7 @@
 @end group
 @end example
 
-@xref{Format with texi2dvi}, and @ref{Preparing for TeX,,Preparing for
+@xref{texi2dvi}, and @ref{Preparing for TeX,,Preparing for
 @TeX{}}, for other ways to format for different paper sizes that do not
 require changing the source file.
 
@@ -15089,7 +15089,7 @@
 margins, you must define a new command in @file{texinfo.tex} (or
 @file{texinfo.cnf}, @pxref{Preparing for TeX,,Preparing for @TeX{}}).
 
-@xref{Format with texi2dvi}, and @ref{Preparing for TeX,,Preparing for
+@xref{texi2dvi}, and @ref{Preparing for TeX,,Preparing for
 @TeX{}}, for other ways to specify @code{@@pagesizes} that do not
 require changing the source file.
 
@@ -15158,7 +15158,7 @@
 magnifications.  Be prepared to experiment.
 
 
-@node PDF Output
+@node texi2pdf
 @section PDF Output
 @cindex PDF output
 
@@ -15166,7 +15166,7 @@
 The simplest way to generate PDF output from Texinfo source is to run
 the convenience script @command{texi2pdf} (or @command{pdftexi2dvi});
 this simply executes the @command{texi2dvi} script with the
-@option{--pdf} option (@pxref{Format with texi2dvi}).  If for some
+@option{--pdf} option (@pxref{texi2dvi}).  If for some
 reason you want to process the document by hand, simply run the
 @command{pdftex} program instead of plain @command{tex}.  That is, run
 @samp{pdftex foo.texi} instead of @samp{tex foo.texi}.
@@ -15270,7 +15270,7 @@
 
 @menu
 * makeinfo advantages::         @code{makeinfo} provides better error checking.
-* Invoking makeinfo::           How to run @code{makeinfo} from a shell.
+* makeinfo::                    How to run @code{makeinfo} from a shell.
 * makeinfo options::            Specify fill-column and other options.
 * Pointer Validation::          How to check that pointers point somewhere.
 * makeinfo in Emacs::           How to run @code{makeinfo} from Emacs.
@@ -15302,7 +15302,7 @@
 @code{makeinfo}.
 
 
-@node Invoking makeinfo
+@node makeinfo
 @subsection Running @code{makeinfo} from a Shell
 @pindex makeinfo
 
@@ -16019,7 +16019,7 @@
                                  located in other directories.
 * Installing Dir Entries::      How to specify what menu entry to add
                                  to the Info directory.
-* Invoking install-info::       @code{install-info} options.
+* install-info::                @code{install-info} options.
 @end menu
 
 
@@ -16330,7 +16330,7 @@
 traditional @command{man} system.
 
 
-@node Invoking install-info
+@node install-info
 @subsection Invoking @command{install-info}
 @pindex install-info
 
@@ -16862,7 +16862,7 @@
 
 In standard Texinfo, node names may not contain @@-commands.
 @command{makeinfo} has an option @option{--commands-in-node-names}
-which partially supports it (@pxref{Invoking makeinfo}), but it is not
+which partially supports it (@pxref{makeinfo}), but it is not
 robust and not recommended.
 
 Thus, @command{makeinfo} also does not fully implement this part of
