diff -ur texinfo-3.12h.old/doc/texinfo.txi texinfo-3.12h/doc/texinfo.txi
--- texinfo-3.12h.old/doc/texinfo.txi	Thu Jul  1 12:35:35 1999
+++ texinfo-3.12h/doc/texinfo.txi	Thu Jul  1 12:39:20 1999
@@ -5,7 +5,7 @@
 @c All text is ignored before the setfilename.
 @setfilename texinfo
 
-@include version.texi
+@include version2.texi
 @settitle Texinfo @value{VERSION}
 
 @c Define a new index for options.
