diff -Nru texinfo-4.5.orig/doc/info-stnd.texi texinfo-4.5/doc/info-stnd.texi
--- texinfo-4.5.orig/doc/info-stnd.texi	Wed Nov  6 01:42:29 2002
+++ texinfo-4.5/doc/info-stnd.texi	Fri Feb 14 10:54:27 2003
@@ -32,10 +32,10 @@
 @end quotation
 @end copying
 
-@dircategory Texinfo documentation system
+@dircategory Texinfo documentation system:
 @direntry
-* info standalone: (info-stnd).            Read Info documents without Emacs.
-* infokey: (info-stnd)Invoking infokey.    Compile Info customizations.
+* info standalone: (info-stnd).		Read Info documents without Emacs.
+* infokey: (info-stnd)Invoking infokey.	Compile Info customizations.
 @end direntry
 
 @titlepage
diff -Nru texinfo-4.5.orig/doc/info.texi texinfo-4.5/doc/info.texi
--- texinfo-4.5.orig/doc/info.texi	Wed Nov  6 01:45:36 2002
+++ texinfo-4.5/doc/info.texi	Fri Feb 14 10:54:27 2003
@@ -35,9 +35,10 @@
 @end quotation
 @end copying
 
-@dircategory Texinfo documentation system
+@dircategory Texinfo documentation system:
 @direntry
-* Info: (info).         How to use the documentation browsing system.
+* Info: (info).				How to use the documentation
+					browsing system
 @end direntry
 
 @titlepage
diff -Nru texinfo-4.5.orig/doc/texinfo.txi texinfo-4.5/doc/texinfo.txi
--- texinfo-4.5.orig/doc/texinfo.txi	Tue Feb  4 16:17:21 2003
+++ texinfo-4.5/doc/texinfo.txi	Fri Feb 14 11:07:13 2003
@@ -55,13 +55,13 @@
 @end quotation
 @end copying
 
-@dircategory Texinfo documentation system
+@dircategory Texinfo documentation system:
 @direntry
-* Texinfo: (texinfo).           The GNU documentation format.
-* install-info: (texinfo)Invoking install-info. Update info/dir entries.
-* texi2dvi: (texinfo)Format with texi2dvi.      Print Texinfo documents.
-* texindex: (texinfo)Format with tex/texindex.  Sort Texinfo index files.
-* makeinfo: (texinfo)Invoking makeinfo.         Translate Texinfo source.
+* Texinfo: (texinfo)			The GNU documentation format
+* install-info: (texinfo)install-info	Update info/dir entries
+* texi2dvi: (texinfo)texi2dvi		Print Texinfo documents
+* texindex: (texinfo)texindex		Sort Texinfo index files
+* makeinfo: (texinfo)makeinfo		Translate Texinfo source
 @end direntry
 
 @c Before release, run C-u C-c C-u C-a (texinfo-all-menus-update with a
@@ -534,8 +534,8 @@
 Formatting and Printing Hardcopy
 
 * Use TeX::                     Use @TeX{} to format for hardcopy.
-* Format with tex/texindex::    How to format with explicit shell commands.
-* Format with texi2dvi::        A simpler way to format.
+* texindex::			How to format with explicit shell commands.
+* texi2dvi::			A simpler way to format.
 * Print with lpr::              How to print.
 * Within Emacs::                How to format and print from an Emacs shell.
 * Texinfo Mode Printing::       How to format and print in Texinfo mode.
@@ -578,7 +578,7 @@
                                  located in other directories.
 * Installing Dir Entries::      How to specify what menu entry to add
                                  to the Info directory.
-* Invoking install-info::       @code{install-info} options.
+* install-info::                @code{install-info} options.
 
 Sample Texinfo Files
 
@@ -607,7 +607,7 @@
 
 Formatting Mistakes
 
-* makeinfo Preferred::          @code{makeinfo} finds errors.
+* makeinfo::			@code{makeinfo} finds errors.
 * Debugging with Info::         How to catch errors with Info formatting.
 * Debugging with TeX::          How to catch errors with @TeX{} formatting.
 * Using texinfo-show-structure::  How to use @code{texinfo-show-structure}.
@@ -2409,7 +2409,7 @@
 you must run the @code{texindex} command after first running the
 @code{tex} typesetting command; and then you must run the @code{tex}
 command again.  Or else run the @code{texi2dvi} command which
-automatically creates indices as needed (@pxref{Format with texi2dvi}).
+automatically creates indices as needed (@pxref{texi2dvi}).
 
 Often, when you are writing a document, you want to typeset and print
 only part of a file to see what it will look like.  You can use the
@@ -4079,8 +4079,8 @@
 
 You need to include the @code{@@set@dots{}contentsaftertitlepage}
 commands early in the document (just after @code{@@setfilename}, for
-example).  We recommend using @command{texi2dvi} (@pxref{Format with
-texi2dvi}) to specify this without altering the source file at all.  For
+example).  We recommend using @command{texi2dvi} (@pxref{(texinfo)texi2dvi})
+to specify this without altering the source file at all.  For
 example:
 @example
 texi2dvi --texinfo=@@setcontentsaftertitlepage foo.texi
@@ -12893,7 +12893,7 @@
 problems with @TeX{}.  If you get macro-related errors when producing
 the printed version of a manual, try expanding the macros with
 @command{makeinfo} by invoking @command{texi2dvi} with the @samp{-E}
-option (@ref{Format with texi2dvi}).
+option (@ref{texi2dvi}).
 
 @itemize @bullet
 @item
@@ -13078,8 +13078,8 @@
 
 @menu
 * Use TeX::                     Use @TeX{} to format for hardcopy.
-* Format with tex/texindex::    How to format with explicit shell commands.
-* Format with texi2dvi::        A simpler way to format.
+* texindex::			How to format with explicit shell commands.
+* texi2dvi::			A simpler way to format.
 * Print with lpr::              How to print.
 * Within Emacs::                How to format and print from an Emacs shell.
 * Texinfo Mode Printing::       How to format and print in Texinfo mode.
@@ -13110,7 +13110,7 @@
 File}).
 
 
-@node Format with tex/texindex
+@node texindex
 @section Format with @code{tex} and @code{texindex}
 @cindex Shell formatting with @code{tex} and @code{texindex}
 @cindex Formatting with @code{tex} and @code{texindex}
@@ -13131,7 +13131,7 @@
 @pindex texindex
 The @code{tex} formatting command itself does not sort the indices; it
 writes an output file of unsorted index data.  (The @code{texi2dvi}
-command automatically generates indices; @pxref{Format with texi2dvi,,
+command automatically generates indices; @pxref{texi2dvi,,
 Format with @code{texi2dvi}}.)  To generate a printed index after
 running the @code{tex} command, you first need a sorted index to work
 from.  The @code{texindex} command sorts indices.  (The source file
@@ -13208,7 +13208,7 @@
 
 @pindex texi2dvi
 Alternatively, it's a one-step process: run @code{texi2dvi}
-(@pxref{Format with texi2dvi}).
+(@pxref{texi2dvi}).
 
 You need not run @code{texindex} each time after you run @code{tex}.  If
 you do not, on the next run, the @code{tex} formatting command will use
@@ -13240,7 +13240,7 @@
 (@pxref{Pointer Validation}).
 
 
-@node Format with texi2dvi
+@node texi2dvi
 @section Format with @code{texi2dvi}
 @pindex texi2dvi @r{(shell script)}
 
@@ -13785,7 +13785,7 @@
 @xref{small}, for information about
 commands that make it easier to produce examples for a smaller manual.
 
-@xref{Format with texi2dvi}, and @ref{Preparing for TeX,,Preparing for
+@xref{texi2dvi}, and @ref{Preparing for TeX,,Preparing for
 @TeX{}}, for other ways to format with @code{@@smallbook} that do not
 require changing the source file.
 
@@ -13815,7 +13815,7 @@
 @end group
 @end example
 
-@xref{Format with texi2dvi}, and @ref{Preparing for TeX,,Preparing for
+@xref{texi2dvi}, and @ref{Preparing for TeX,,Preparing for
 @TeX{}}, for other ways to format for different paper sizes that do not
 require changing the source file.
 
@@ -13861,7 +13861,7 @@
 margins, you must define a new command in @file{texinfo.tex} (or
 @file{texinfo.cnf}, @pxref{Preparing for TeX,,Preparing for @TeX{}}).
 
-@xref{Format with texi2dvi}, and @ref{Preparing for TeX,,Preparing for
+@xref{texi2dvi}, and @ref{Preparing for TeX,,Preparing for
 @TeX{}}, for other ways to specify @code{@@pagesizes} that do not
 require changing the source file.
 
@@ -14751,7 +14751,7 @@
                                  located in other directories.
 * Installing Dir Entries::      How to specify what menu entry to add
                                  to the Info directory.
-* Invoking install-info::       @code{install-info} options.
+* install-info::                @code{install-info} options.
 @end menu
 
 
@@ -14998,7 +14998,7 @@
 @@dircategory Texinfo documentation system
 @@direntry
 * Texinfo: (texinfo).           The GNU documentation format.
-* install-info: (texinfo)Invoking install-info. @dots{}
+* install-info: (texinfo)install-info. @dots{}
 @dots{}
 @@end direntry
 @end smallexample
@@ -15009,7 +15009,7 @@
 INFO-DIR-SECTION Texinfo documentation system
 START-INFO-DIR-ENTRY
 * Texinfo: (texinfo).           The GNU documentation format.
-* install-info: (texinfo)Invoking install-info. @dots{}
+* install-info: (texinfo)install-info. @dots{}
 @dots{}
 END-INFO-DIR-ENTRY
 @end smallexample
@@ -15052,7 +15052,7 @@
 traditional @command{man} system.
 
 
-@node Invoking install-info
+@node install-info
 @subsection Invoking @command{install-info}
 @pindex install-info
 
@@ -17675,7 +17675,7 @@
 command and you can use the @kbd{M-x Info-validate} command.@refill
 
 @menu
-* makeinfo Preferred::          @code{makeinfo} finds errors.
+* makeinfo::			@code{makeinfo} finds errors.
 * Debugging with Info::         How to catch errors with Info formatting.
 * Debugging with TeX::          How to catch errors with @TeX{} formatting.
 * Using texinfo-show-structure::  How to use @code{texinfo-show-structure}.
@@ -17683,7 +17683,7 @@
 * Running Info-Validate::       How to find badly referenced nodes.
 @end menu
 
-@node makeinfo Preferred, Debugging with Info, Catching Mistakes, Catching Mistakes
+@node makeinfo, Debugging with Info, Catching Mistakes, Catching Mistakes
 @ifinfo
 @heading @code{makeinfo} Find Errors
 @end ifinfo
@@ -17702,7 +17702,7 @@
 cannot use @code{makeinfo}, or your problem is very puzzling, then you
 may want to use the tools described in this appendix.@refill
 
-@node Debugging with Info, Debugging with TeX, makeinfo Preferred, Catching Mistakes
+@node Debugging with Info, Debugging with TeX, makeinfo, Catching Mistakes
 @comment  node-name,  next,  previous,  up
 @section Catching Errors with Info Formatting
 @cindex Catching errors with Info formatting
